homeassistant/components/dpi4relay/__init__.py

```diff
@@ -0,0 +1,3 @@
+"""Support for DockerPi 4 Channel Relay expansion board."""
+
+DOMAIN = "dpi4relay"
```

Comment:
Please move this constant to either a separate const.py file or to switch.py. I would recommend the latter since it is only one constant.